docs/evm_compatibility/overview.md

```diff
@@ -45,6 +46,8 @@ Several Solana-specific differences also impact smart contract development.
 ### Upper limit on number of Accounts
 Neon EVM uses [Solana Transaction V2](https://docs.solana.com/proposals/transactions-v2): limiting the maximum number of accounts used in a single transaction to 64. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions. 
```

Comment:
There does not seem to be a Solana Transaction V2!!
Semen recons it should be this link https://docs.solana.com/developing/versioned-transactions
as in legacy was v1 and current, v0 == 2?!
WDYT @YYYurchenko ?

Reply:
It seems that V2 was the old name which Solana renamed to v0. Please align it with Solana docs
https://docs.solana.com/developing/lookup-tables